package.json

```diff
@@ -41,52 +45,40 @@
   },
   "homepage": "https://github.com/open-formulieren/formio-renderer#readme",
   "peerDependencies": {
-    "formiojs": "~4.13.0 || ~4.14.0",
-    "react": "^17.0.2"
+    "react": "^18.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.21.0",
-    "react-formio": "^4.3.0",
-    "@storybook/addon-actions": "^6.5.16",
-    "@storybook/addon-essentials": "^6.5.16",
-    "@storybook/addon-interactions": "^6.5.16",
-    "@storybook/addon-links": "^6.5.16",
-    "@storybook/builder-webpack5": "^6.5.16",
-    "@storybook/jest": "^0.0.10",
-    "@storybook/manager-webpack5": "^6.5.16",
-    "@storybook/react": "^6.5.16",
-    "@storybook/test-runner": "^0.10.0",
-    "@storybook/testing-library": "^0.0.13",
-    "@trivago/prettier-plugin-sort-imports": "^4.1.1",
-    "@types/react": "^18.0.28",
-    "@types/webpack": "^5.28.1",
-    "@typescript-eslint/eslint-plugin": "^5.59.0",
-    "@typescript-eslint/parser": "^5.59.0",
-    "babel-loader": "^8.3.0",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.7.0",
-    "eslint-config-standard": "^14.1.0",
-    "eslint-config-standard-react": "^9.2.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-prettier": "^3.1.1",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.17.0",
-    "eslint-plugin-standard": "^4.0.1",
-    "formiojs": "~4.13.0",
+    "@babel/core": "^7.24.0",
+    "@formatjs/ts-transformer": "^3.12.0",
+    "@storybook/addon-actions": "^8.0.0",
+    "@storybook/addon-essentials": "^8.0.0",
+    "@storybook/addon-interactions": "^8.0.0",
+    "@storybook/addon-links": "^8.0.0",
+    "@storybook/client-api": "^7.6.17",
```

Comment:
is this dependency still needed? It's a lower major version than the main SB install (v8) so I expect problems.

You can run `npm why @storybook/client-api` to get more information :)